server/routes/request-schema.ts

```diff
@@ -0,0 +1,14 @@
+import { check } from 'express-validator'
+
+const regExp = /^[A-Za-z0-9]*$/
+const schema = [
+  check('prisonerNumber').not().isEmpty({ ignore_whitespace: true }),
+  check('prisonerNumber').isLength({ min: 1 }),
```

Comment:
I think it's already definitely length > 1 if it is not empty

Reply:
removed the unwanted check!